app/routes/bestuurseenheid/zitting/uittreksels/detail/index.js

```diff
@@ -2,12 +2,17 @@ import Route from '@ember/routing/route';
 
 export default class BestuurseenheidZittingUittrekselsDetailIndexRoute extends Route {
   async model() {
-    const uittreksel = this.modelFor('bestuurseenheid.zitting.uittreksels.detail');
+    const uittreksel = this.modelFor(
+      'bestuurseenheid.zitting.uittreksels.detail'
+    );
     const publication = uittreksel.get('publication');
+    const zitting = this.modelFor('bestuurseenheid.zitting');
+    const bestuursorgaan = await zitting.bestuursorgaan;
+    await bestuursorgaan.isTijdsspecialisatieVan;
```

Comment:
That's the same setup again :smile: if all pages use this, it makes sense to move it to the parent route.